collect.py

```diff
@@ -95,6 +95,37 @@ def get_all_attributes(driver, child):
     return child_attributes
 
 
+def check_attributes(driver, elem_attributes, elem=None):
```

Comment:
Rename this function to something like `get_elements_with_attributes` and make it return a list of elements matching the attributes.
The callers will check the length of the list and make sure it is 1.